src/payers/hypha_payer.cpp

```diff
@@ -9,7 +9,7 @@
 #include <common.hpp>
 
 // 3 year expiry
-#define EXPIRY_MICROSECONDS eosio::microseconds(3 * 365 * 24 * 60 * 60 * 1000000)
```

Comment:
This causes an overflow error since the final value cannot be stored in a 32 bytes variable (default for **int** on this architecture). Using eosio built-in `days` function we avoid having to convert to int64_t literals.